components/picker/index.css

```diff
@@ -491,3 +491,10 @@
 		color: var(--highcontrast-picker-content-color-disabled, var(--mod-picker-font-color-disabled, var(--spectrum-picker-font-color-disabled)));
 	}
 }
+
+/* Fixed menu height */
+.spectrum-Picker--menu-height-fixed {
+	& + .spectrum-Popover.is-open {
```

Comment:
Do we need the `.is-open` here? Can't the popover `max-block-size` be defined independently of being open?

Reply:
For specificity, I think it's necessary. It also gives other devs more context.